tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -1989,5 +1989,9 @@ def test_get_orderby_clauses():
         with pytest.raises(MlflowException, match=match):
             _get_orderby_clauses(["tag.t", "tag.t"], session)
 
+        # test that "=" is used rather than "is" when comparing to True
```

Comment:
Please move this section to the end of the test to not break the context on exception testing on L1996.